src/hooks/useAppHooks.js

```diff
@@ -46,7 +168,8 @@ export function useVaultBalance(installedApps, token, timeout = 1000) {
           const vaultBalance = await vaultContract.balance(token.id)
 
           if (!cancelled) {
-            setVaultBalance(vaultBalance)
+            // Contract value is bn.js so we need to convert it to bignumber.js
```

Comment:
is always like this? doesn't the contract calls return a string?

Reply:
Thought it did return a string but it just returns a `BN`